src/components/Input/index.tsx

```diff
@@ -31,7 +33,14 @@ function Input({
 }
 
 
+Input.Checkbox = Checkbox;
 Input.Icon = InputIcon;
```

Comment:
@lawren, since the `Input` component is just a wrapper anyway, what do you think about doing away with it and just exporting the inputs directly from this file?

Reply:
I'm cool with it. It'd probably fit convention a bit better.